resources/js/photoinit.js

```diff
@@ -67,20 +67,21 @@ function initPhotoSwipeFromDOM (gallerySelector) {
             shareEl: false,
             zoomEl: false,
             fullscreenEl: false,
+	    tapToToggleControls: false

```

Comment:
Space issue. You can run `yarn eslint` to check for js errors.

Reply:
Great - thanks for the hint, will explore further.